core/geo/geoUtils.js

```diff
@@ -51,23 +56,68 @@ export const findSrsByCodeOrName = (codeOrName, limit = 200) =>
 
 const getSrsByCode = (code) => srsByCode[code]
 
+const toLatLon = (point) => {
```

Comment:
toLatLng (by convention it's use Lng)

Reply:
lat lon or lat long is more common, I vote for "lat long"